### PR TITLE
use classic flow for v1 test

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## PR Checklist

- [ ] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

**Review by shuowu-okta — APPROVED** [review]

File: test/app/src/TestApp.ts [hunk 1]
```diff
@@ -370,7 +370,7 @@ export default class TestApp {
     const authClient = oktaSign.authClient;
 
     // email verify callback
-    if (authClient.idx.isEmailVerifyCallback(window.location.search)) {
+    if (authClient.idx?.isEmailVerifyCallback(window.location.search)) {
```
Comment (shuowu-okta) [inline test/app/src/TestApp.ts:373]: nit: ? -> !

File: test/e2e/steps/given.ts [hunk 2]
```diff
@@ -51,7 +51,8 @@ const basicConfig = {
   el: '#okta-login-container',
   authParams: { pkce:false },
   clientId: WIDGET_WEB_CLIENT_ID,
-  scopes: ['openid', 'email', 'profile']
```
Comment (shuowu-okta) [inline test/e2e/steps/given.ts:54]: nit: rename `basicConfig` to be something specific about authn flow, as it's not the default one anymore

**Review by shuowu-okta — APPROVED** [review]

File: test/app/src/TestApp.ts (hunk 1 above)
Comment (oprav) [reply test/app/src/TestApp.ts:373]: `.idx` doesn't seem to be guaranteed to exist when using `classic` auth-js bundle

Comment (oleksandrpravosudko-okta) [reply test/app/src/TestApp.ts:373]: `.idx` doesn't seem to be guaranteed to exist when using 'classic' auth-js bundle

**Pull request merged**